src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java

```diff
@@ -18,10 +18,10 @@
 package org.jitsi.impl.protocol.xmpp;
 
 import net.java.sip.communicator.impl.protocol.jabber.*;
+import org.jitsi.utils.logging.*;
```

Comment:
can you check with this change that the XMPP packets logging works as expected? see a comment in logging.properties on how to enable or ping me if you have problems

Reply:
Confirmed, XMPP logging works as before